package.json

```diff
@@ -15,6 +15,7 @@
     "build:plugins": "turbo run build --filter=...{./plugins/*}",
```

Comment:
You will need to update the build scripts. I would suggest using a scope filter. https://turbo.build/repo/docs/core-concepts/monorepos/filtering#scopes

Reply:
Good catch, I'll update that & check for other references to the plugins directory.